jms-reactive-client/src/main/java/edu/psu/activemq/exception/UnableToProcessMessageException.java

```diff
@@ -29,9 +29,27 @@
     DROP
   }
   
+  public enum RetryStyle {
+    LINEAR,
+    EXPONENTIAL
+  }
+  
+  
   int retryWait = 0;
   HandleAction handleAction = HandleAction.ERROR;
   
+  @Getter
```

Comment:
It might be nice to have a retry interface so that the conventional retry strategies can simply be plugged in.